rossum/hook.py

```diff
@@ -174,6 +194,12 @@ def change_command(
             data["config"] = config
         if sideload:
             data["sideload"] = list(sideload)
+        if token_owner:
+            data["token_owner"] = f"{rossum.url}/users/{token_owner}"
+        if run_after:
+            data["run_after"] = [f"{rossum.url}/hooks/{hook_id}" for hook_id in run_after]
+        if test:
+            data["test"] = test
```

Comment:
```suggestion
            data["test"] = json.loads(test)
```
I believe if I understand the API correctly - and to fit the type hints. (And analogously in `create_command()`.)